lib/readline.js

```diff
@@ -487,7 +498,7 @@ Interface.prototype._normalWrite = function(b) {
 };
 
 Interface.prototype._insertString = function(c) {
-  if (this.cursor < this.line.length) {
+  if (this.cursor < this.line.length || this.postprocessor) {
```

Comment:
I am not certain why the postprocessor should always use this branch. Please add a comment, as it does not seem obvious.